test-scripts/coverage.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+echo "" > coverage.txt
+
+for d in $(go list ./... | grep -v vendor); do
```

Comment:
What does "d" stand for here? Can we use a more descriptive name?

Reply:
d in this case would be directory, or whatever go list ./... happens to return